### PR TITLE
Assessment list view

### PR DESCRIPTION
**Pull request by thematthopkins**
#### Overview
Changes include a contextual component `expandable-collection` and component `overflow-list`. The first a wrapper around the Materialize popout functionality, the latter is the overflow menu that we have in several locations.

The other major change includes mock responses now include unique ids for `section` and `group` models.

#### Notable Jawns
Take a look at the `disable-bubbling` helper; which is required make the overflow menu possible on an expandable header. It's the only way to prevent the event propagation in an Ember way without doing anything 🍌 .

There's also a minor change to `cleanup` after `ember-try`. Hopefully this will fix the auto-deployment issues.

### CODE REVIEW

File: addon/models/group.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ export default Model.extend({
   summaryTotalQuestions: attr('number'),
   progress: attr('number'),
 
-  assessment: belongsTo('assessment', { async: true }),
+  assessment: hasMany('assessment', { async: true }),
```
Comment (kikemoto) [inline addon/models/group.js]: @thematthopkins Can you help me understand this relationship? Using mocked test data as an example, how can 1 group id: `1-quick-impact-assessment/impact-business-models` have many assessments?

File: addon/templates/components/overflow-list.hbs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div class="more" onclick={{disable-bubbling (action "showMenu")}}>
```
Comment (kikemoto) [inline addon/templates/components/overflow-list.hbs]: @thematthopkins The comments on the PR mention that `expandable-collection` and this component are contextual. Is that correct, is this component contextual?

File: addon/models/group.js (hunk 1 above)
Comment (thematthopkins) [inline addon/models/group.js]: @kikemoto - Great point.  @jonpitch does the work around unique group IDs make this a `belongsTo`?

File: addon/templates/components/overflow-list.hbs (hunk 2 above)
Comment (jonpitch) [inline addon/templates/components/overflow-list.hbs]: @kikemoto the overview is incorrect, `overflow-list` is not contextual.

File: addon/models/group.js (hunk 1 above)
Comment (jonpitch) [inline addon/models/group.js]: @kikemoto @thematthopkins I think that's correct, it should probably be a `belongsTo`

**Comment by thematthopkins** [comment]
@kikemoto + @jonpitch - running into some failed tests when pointing the impact-app to this branch.  Investigating now...

**Comment by thematthopkins** [comment]
@kikemoto + @jonpitch - nevermind - I was pointing to `master` `impact-app`.  Re-testing on `impact-app` `assessment-list-view`.

**Pull request merged**